packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-button.tsx

```diff
@@ -115,20 +122,25 @@ const CFDCompareAccountsButton = observer(({ trading_platforms, is_demo }: TComp
     );
 
     const onClickAdd = () => {
-        setAppstorePlatform(trading_platforms.platform as string);
```

Comment:
Having all these logic in a file called cfd-compare-account-button is totally an overkill, can we not refactor this to use hooks for the logic?

Reply:
Made the hook and added all these things in that for a little cleaner version of component.